### PR TITLE
Add Hebrew keymap aliases

### PR DESCRIPTION
**Pull request by spidey3**
## Description

Add a keymap_extras mapping for Hebrew, to make construction of keymaps 
for Hebrew language keyboards easier.

## Types of Changes

- [x] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: quantum/keymap_extras/keymap_hebrew.h [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+#define IL_DALET   KC_S    // ד
+#define IL_GIMEL   KC_D    // ג
+#define IL_KAF     KC_F    // כ
+#define IL_AYIN    KC_G    // ע
+#define IL_YOD     KC_H    // י
+#define IL_CHET    KC_J    // ח
+#define IL_LAMED   KC_K    // ל
+#define IL_KAF_S   KC_L    // ך
+#define IL_PAY_S   KC_SCLN // ף
+#define IL_COMM    KC_QUOT // '
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:76]:
```suggestion
#define IL_SHIN KC_A    // ש
#define IL_DALT KC_S    // ד
#define IL_GIML KC_D    // ג
#define IL_KAF  KC_F    // כ
#define IL_AYIN KC_G    // ע
#define IL_YOD  KC_H    // י
#define IL_HET  KC_J    // ח
#define IL_LAMD KC_K    // ל
#define IL_FKAF KC_L    // ך
#define IL_FPE  KC_SCLN // ף
#define IL_COMM KC_QUOT // ,
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#define IL_QOF     KC_E    // ק
+#define IL_RESH    KC_R    // ר
+#define IL_ALEF    KC_T    // א
+#define IL_TET     KC_Y    // ט
+#define IL_VAV     KC_U    // ו
+#define IL_NUN_S   KC_I    // ן
+#define IL_MEM_S   KC_O    // ם
+#define IL_PAY     KC_P    // פ
+#define IL_RBRC    KC_LBRC // [
+#define IL_LBRC    KC_RBRC // ]
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:64]:
```suggestion
#define IL_SLSH KC_Q    // /
#define IL_QUOT KC_W    // '
#define IL_QOF  KC_E    // ק
#define IL_RESH KC_R    // ר
#define IL_ALEF KC_T    // א
#define IL_TET  KC_Y    // ט
#define IL_VAV  KC_U    // ו
#define IL_FNUN KC_I    // ן
#define IL_FMEM KC_O    // ם
#define IL_PE   KC_P    // פ
#define IL_RBRC KC_LBRC // ]
#define IL_LBRC KC_RBRC // [
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+#define IL_ZAYIN   KC_Z    // ז
+#define IL_SAMECH  KC_X    // ס
+#define IL_BET     KC_C    // ב
+#define IL_HEH     KC_V    // ה
+#define IL_NUN     KC_B    // נ
+#define IL_MEM     KC_N    // מ
+#define IL_TSADE   KC_M    // צ
+#define IL_TAV     KC_COMM // ת
+#define IL_TSADE_S KC_DOT  // ץ
+#define IL_DOT     KC_SLSH // .
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:87]:
```suggestion
#define IL_ZAYN KC_Z    // ז
#define IL_SMKH KC_X    // ס
#define IL_BET  KC_C    // ב
#define IL_HE   KC_V    // ה
#define IL_NUN  KC_B    // נ
#define IL_MEM  KC_N    // מ
#define IL_TSDI KC_M    // צ
#define IL_TAV  KC_COMM // ת
#define IL_FTSD KC_DOT  // ץ
#define IL_DOT  KC_SLSH // .
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+#define IL_DLR  S(IL_4)       // $
+#define IL_PERC S(IL_5)       // %
+#define IL_CIRC S(IL_6)       // ^
+#define IL_AMPR S(IL_7)       // &
+#define IL_ASTR S(IL_8)       // *
+#define IL_RPRN S(IL_9)       // )
+#define IL_LPRN S(IL_0)       // (
+#define IL_UNDS S(IL_MINS)    // _
+#define IL_PLUS S(IL_EQL)     // +
+#define IL_PIPE S(IL_BSLS)    // |
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:116]:
```suggestion
#define IL_TILD S(IL_SCLN) // ~
#define IL_EXLM S(IL_1)    // !
#define IL_AT   S(IL_2)    // @
#define IL_PND  S(IL_3)    // #
#define IL_DLR  S(IL_4)    // $
#define IL_PERC S(IL_5)    // %
#define IL_CIRC S(IL_6)    // ^
#define IL_AMPR S(IL_7)    // &
#define IL_ASTR S(IL_8)    // *
#define IL_RPRN S(IL_9)    // )
#define IL_LPRN S(IL_0)    // (
#define IL_UNDS S(IL_MINS) // _
#define IL_PLUS S(IL_EQL)  // +
#define IL_PIPE S(IL_BSLS) // |
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+// Row 2
+#define IL_RCBR S(IL_RBRC)    // }
+#define IL_LCBR S(IL_LBRC)    // {
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:119]:
```suggestion
#define IL_RCBR S(IL_RBRC) // }
#define IL_LCBR S(IL_LBRC) // {
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+// Row 3
+#define IL_COLN S(IL_PAY_S)   // :
+#define IL_DQUO S(IL_COMM)    // "
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:122]:
```suggestion
#define IL_COLN S(IL_FPE)  // :
#define IL_DQUO S(IL_COMM) // "
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+// Row 4
+#define IL_RABK S(IL_TAV)     // >
+#define IL_LABK S(IL_TSADE_S) // <
+#define IL_QUES S(IL_DOT)     // ?
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:126]:
```suggestion
#define IL_RABK S(IL_TAV)  // >
#define IL_LABK S(IL_FTSD) // <
#define IL_QUES S(IL_DOT)  // ?
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+ */
+
+// Row 1
+#define IL_EURO   ALGR(IL_3)    // €
+#define IL_SHEKEL ALGR(IL_4)    // ₪
+// Row 2
+#define IL_DBLVAV ALGR(IL_VAV)  // װ
+// Row 3
+#define IL_DBLYOD ALGR(IL_YOD)  // ײ
+#define IL_VAVYOD ALGR(IL_CHET) // ױ
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:149]:
```suggestion
#define IL_EURO ALGR(IL_3)    // €
#define IL_SHKL ALGR(IL_4)    // ₪
// Row 2
#define IL_DVAV ALGR(IL_VAV)  // װ
// Row 3
#define IL_DYOD ALGR(IL_YOD)  // ײ
#define IL_VYOD ALGR(IL_HET)  // ױ
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+/*
+ * NOTE:
+ * There is no uniform method for entering Niqqud, so 
+ * we do not attempt to provide a mapping here.
+ */
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:155]:
```suggestion
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+ * │ ; │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ \ │   │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤
+ * │     │ / │ ' │ פ │ ם │ ן │ ו │ ט │ א │ ר │ ק │ ] │ [ │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘     │
+ * │      │ ף │ ך │ ל │ ח │ י │ ע │ כ │ ג │ ד │ ש │ , │        │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤
+ * │        │ ץ │ ת │ צ │ מ │ נ │ ה │ ב │ ס │ ז │ . │          │
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:33]: Please make sure all of the internal whitespace in these box drawings are NBSPs.

File: quantum/keymap_extras/keymap_hebrew.h [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:36]:
```suggestion
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+ * │     │   │   │   │   │   │   │ װ │   │   │   │   │   │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘     │
+ * │      │   │   │   │   │   │ ױ │ ײ │   │   │   │   │        │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤
+ * │    │   │   │   │   │   │   │   │   │   │   │   │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:141]:
```suggestion
```

File: quantum/keymap_extras/keymap_hebrew.h (hunk 10 above)
Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h]: Done.

**Comment by spidey3** [comment]
@fauxpark I find it amusing that even here in QMK there are arguments about how to correctly transliterate Hebrew into English characters :)

**Comment by fauxpark** [comment]
I'm just basing them off the Unicode names, and keeping them max 4 characters.

**Comment by spidey3** [comment]
> I'm just basing them off the Unicode names, and keeping them max 4 characters.
I was using spellings from another source (Wikipedia), but I agree, the Unicode naming is better.

File: quantum/keymap_extras/keymap_hebrew.h [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define IL_EURO ALGR(IL_3)    // €
+#define IL_SHKL ALGR(IL_4)    // ₪
+// Row 2
+#define IL_DVAV ALGR(IL_VAV)  // װ
+// Row 3
+#define IL_DYOD ALGR(IL_YOD)  // ײ
+#define IL_VYOD ALGR(IL_HET)  // ױ
```
Comment (noroadsleft) [inline quantum/keymap_extras/keymap_hebrew.h:147]: I can't make any sense of why, but when I test this on my system (Windows 8.1), these three characters are not on the keys assigned here.

- `װ` is on AltGr+QWERTY `Y`
- `ײ` is on AltGr+QWERTY `H`
- `ױ` is on AltGr+QWERTY `G`

The assignments here match what's on Wikipedia, so I'm not sure what to do with this knowledge.

Comment (fauxpark) [reply quantum/keymap_extras/keymap_hebrew.h:147]: Same on Windows 10. They should match what the OSK displays IMO.
Also, I see a bunch of other characters (mostly diacritics) that should probably be added too, for completeness.

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h:147]: > Same on Windows 10. They should match what the OSK displays IMO.

DVAV, DYOD, and VYOD are principally used when typing Yiddish in Hebrew lettering. You may not be using the same Windows Hebrew keyboard layout (There are multiple options - see below). They work on Mac, Linux and ChromeOS as I present them here. If there is no uniformity in how they work, then perhaps they should be removed (few use them, as they are just shortcuts for letter combinations that are common).

> Also, I see a bunch of other characters (mostly diacritics) that should probably be added too, for completeness.

This was exactly the subject of the comment that you asked me to remove:
```c
/*
 * There is no uniform method for entering Niqqud, so 
 * we do not attempt to provide a mapping here.
*/
```
TL;DR: _Niqqud_ (plural _niqqudot_) is the (transliterated) Hebrew word for diacritics, and is, put succinctly, a can of worms that I don't think QMK should open under any circumstances, especially given that they are almost never used when typing, and there is no agreement on a standard layout. 

Longer explanation:

Background:
- Hebrew text is an [Abjad](https://en.wikipedia.org/wiki/abjad) writing system, where each glyph stands for a consonant, leaving it to readers to infer or otherwise supply the appropriate vowels.
- _Niqqud_ is used (even when handwriting) only in cases where exact pronunciation is deemed critical, e.g. in prayerbooks or modern transcriptions of scripture. Interestingly, ancient Hebrew artifacts, and even newly created Torah scrolls, do not have _niqqudot_.
- When moveable type was the order of the day, printing was already a laborious process, so the addition of diacritics to a text wasn't really a big deal, and many Hebrew texts were printed with _niqqudot_. 
- But typewriters were a different story, and people typing business letters are lazy, so even if the typewriter supported _niqqud_ (many didn't), it generally was not used.

The end result is that Modern Hebrew is almost never written or printed with any use of diacritics. For instance see [the front page of Ha'aretz](https://www.haaretz.co.il/), a leading Israeli newspaper.  Consequently, no good standard has evolved for entering _niqqudot_. The [wikipedia article](https://en.wikipedia.org/wiki/Hebrew_keyboard#Niqqud) has a table showing the most common input methods, but there are far more than the two methods shown there.

So, I propose to ignore the existence of _niqqud_ in `keymap_hebrew.h`. Adding them will only cause confusion.

It should also be pointed out that are at least five different Hebrew keyboard mappings in common use, and I have only implemented the [SII](https://en.wikipedia.org/wiki/Standards_Institute_of_Israel) standard:

1. [Hebrew](http://kbdlayout.info/kbdheb)
2. [Hebrew (SII Standard)](http://kbdlayout.info/kbdhebl3) ← [implemented in `keymap_hebrew.h`]
3. [Hebrew (Phonetic)](https://egarc.ku.edu/installing-hebrew-ku-homophonic-keyboard-windows-7)
4. [Biblical Hebrew (Tiro)](https://www.sbl-site.org/Fonts/BiblicalHebrewTiroManual.pdf)
5. [Biblical Hebrew (SIL)](https://www.sbl-site.org/Fonts/BiblicalHebrewSILManual.pdf)

Comment (fauxpark) [reply quantum/keymap_extras/keymap_hebrew.h:147]: These are the only two choices of keyboard I have when I add the Hebrew language. The SII layout was the default.
image
They seem to be identical except for the AltGr symbols.
Hebrew (Standard):
image
Hebrew:
image

The system I've been going with for these headers is to have `keymap_*.h` be the default Windows one for that language, and `keymap_*_osx_[ansi|iso].h` for the macOS ones. So, I think we should at least fix those three characters, and include the div/mul symbols as well as anything else here that is not specifically Niqqud.

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h:147]: I am glad that Windows now has SII as the default; that wasn't the case the last time I used Windows extensively for Hebrew typing (about 10 years ago).

There is an interesting phenomenon here, which may guide us. The majority of daily Hebrew speakers live in the State of Israel (about 6.5 million); but there are lots of folks in the rest of the world who use Hebrew only for liturgical purposes (about 7.6 million), using English (or some other latin-alphabet language) as their primary language. Also there are many who use Yiddish daily (about 0.6 million). I note, however, that those who use Hebrew for liturgical purposes are less likely to type in Hebrew regularly. For instance, I live in the U.S.A.; English is my primary language, but I can read and write both Modern and Biblical Hebrew, and I'm reasonably conversant. I think I type in Hebrew perhaps once or twice a month, mostly when communicating with friends in Israel (in Modern Hebrew) or doing things related to my synagogue (using Biblical Hebrew).

Given the above, I agree with your suggested approach. Shekel, Euro are already in the map; I will add div, mul and degree.

I worry about the parens and brackets though; your images differ from the behavior that I see. Can you please check behavior when typing on a keyboard? When I test on Mac and ChromeOS, I see them reversed when using Hebrew (an RTL language). Shift-9 is "open paren", and Shift-0 is "close paren", and similar for the bracket / brace keys...

Comment (fauxpark) [reply quantum/keymap_extras/keymap_hebrew.h:147]: Yep, they're reversed when actually typing. Dunno why it's not in the OSK...

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h:147]: Oops - looks like there are no keycodes defined in keycode.h for `◦`, `×` or `÷`, so I won't do those.

Comment (fauxpark) [reply quantum/keymap_extras/keymap_hebrew.h:147]: Doesn't matter: https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_us_international.h#L198

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h:147]: ...and it looks like װ ,ױ ,ײ are already correct, so I don't think there is anything more to be done.

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h]: Fixed IL_DVAV, IL_DYOD and IL_VYOD, which were incorrect after all.

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h]: > Doesn't matter:
Added ◦, × and ÷.

File: quantum/keymap_extras/keymap_hebrew.h [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+
+#pragma once
+
+#include "keymap.h"
+
+// clang-format off
+
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+ * │ ; │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ \ │   │
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:25]: Aaand I just noticed these box drawings are inconsistent. Do Hebrew keyboards still use BAE or is it usually the standard ISO enter these days?

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h:25]: That's a good question.

Comment (spidey3) [reply quantum/keymap_extras/keymap_hebrew.h]: Yup, seems like the diagram at Wikipedia is out of date vs. the real world. 
I have updated to ISO Enter, and switched IL_BSLS to use KC_NUHS, which is similar to Greek, Latvian and Lithuanian QWERTY which have backslash + pipe on that key. I've tested on Mac and ChromeOS.

File: quantum/keymap_extras/keymap_hebrew.h [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+#define IL_FTSD KC_DOT  // ץ
+#define IL_DOT  KC_SLSH // .
+
+/* Shifted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ~ │ ! │ @ │ # │ $ │ % │ ^ │ & │ * │ ) │ ( │ _ │ + │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │   │   │   │   │   │   │   │   │   │   │ } │ { │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │   │   │   │   │   │ : │ " │   │    │
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:94]:
```suggestion
 * │      │   │   │   │   │   │   │   │   │   │ : │ " │ | │    │
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 16]
```diff
@@ -0,0 +1,151 @@
...
+#define IL_DLR  S(IL_4)    // $
+#define IL_PERC S(IL_5)    // %
+#define IL_CIRC S(IL_6)    // ^
+#define IL_AMPR S(IL_7)    // &
+#define IL_ASTR S(IL_8)    // *
+#define IL_RPRN S(IL_9)    // )
+#define IL_LPRN S(IL_0)    // (
+#define IL_UNDS S(IL_MINS) // _
+#define IL_PLUS S(IL_EQL)  // +
+#define IL_PIPE S(IL_BSLS) // |
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:115]:
```suggestion
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 17]
```diff
@@ -0,0 +1,151 @@
...
+// Row 2
+#define IL_RCBR S(IL_RBRC) // }
+#define IL_LCBR S(IL_LBRC) // {
+// Row 3
+#define IL_COLN S(IL_FPE)  // :
+#define IL_DQUO S(IL_COMM) // "
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:121]:
```suggestion
#define IL_DQUO S(IL_COMM) // "
#define IL_PIPE S(IL_BSLS) // |
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 18]
```diff
@@ -0,0 +1,151 @@
...
+// Row 4
+#define IL_RABK S(IL_TAV)  // >
+#define IL_LABK S(IL_FTSD) // <
+#define IL_QUES S(IL_DOT)  // ?
+
+/* AltGr symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │   │   │   │ € │ ₪ │   │   │   │   │   │   │   │   │       │
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:129]:
```suggestion
 * │   │   │   │ € │ ₪ │ ° │   │   │ × │   │   │   │   │       │
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 19]
```diff
@@ -0,0 +1,151 @@
...
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │   │   │   │   │   │ װ │   │   │   │   │   │   │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │ ױ │ ײ │   │   │   │   │   │   │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │   │   │   │   │   │   │   │   │   │   │   │          │
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:135]:
```suggestion
 * │    │   │   │   │   │   │   │   │   │   │   │ ÷ │          │
```

File: quantum/keymap_extras/keymap_hebrew.h [hunk 20]
```diff
@@ -0,0 +1,151 @@
...
+#define IL_LABK S(IL_FTSD) // <
+#define IL_QUES S(IL_DOT)  // ?
+
+/* AltGr symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │   │   │   │ € │ ₪ │ ° │   │   │ × │   │   │   │   │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │   │   │   │   │   │ װ │   │   │   │   │   │   │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │ ױ │ ײ │   │   │   │   │   │   │    │
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_hebrew.h:133]:
```suggestion
 * │      │   │   │   │   │ ײ │ ױ │   │   │   │   │   │   │    │
```

**Comment by noroadsleft** [comment]
Thanks!
